python/pyspark/sql/functions.py

```diff
@@ -5146,26 +5146,27 @@ def log(arg1: Union["ColumnOrName", float], arg2: Optional["ColumnOrName"] = Non
 
     Examples
     --------
-    >>> df = spark.createDataFrame([10, 100, 1000], "INT")
-    >>> df.select(log(10.0, df.value).alias('ten')).show() # doctest: +SKIP
-    +---+
-    |ten|
-    +---+
```

Comment:
the previous values are:
```
+------------------+
|               1.0|
|               2.0|
|2.9999999999999996|
+------------------+
```

I think the previous doctest was skipped due to the the computation error.

With new input values (2,4,8) and new logarithm 2.0, the output happens to be accurate.
So make this change.